cli/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 12.9.0
+
+_Released 03/28/2023 (PENDING)_
+
+**Bugfixes:**
+
+ - Dropped the `incremental` esbuild option from our Vite dev server configuration after Vite 4.2.0. Fixes [#26138](https://github.com/cypress-io/cypress/issues/26138).
```

Comment:
- Dropped the `incremental` esbuild option from our Vite dev server configuration after Vite 4.2.0. Fixes [#26138](https://github.com/cypress-io/cypress/issues/26138).
```suggestion
 - Fixed a compatibility issue so that component test projects can use [Vite](https://vitejs.dev/) version 4.2.0 and greater. Fixes [#26138](https://github.com/cypress-io/cypress/issues/26138).
```